sensor.py

```diff
@@ -15,13 +15,16 @@
                 "subclasses": {
                     "Outside_Air_Grains_Sensor": {
                         "tags": [ TAG.Outside, TAG.Air, TAG.Grains, TAG.Sensor ],
+                        "substances": [ [ BRICK.measures, BRICK.Outside_Air ], [ BRICK.measures, BRICK.Grains ], ],
                     },
                     "Return_Air_Grains_Sensor": {
                         "tags": [ TAG.Return, TAG.Air, TAG.Grains, TAG.Sensor ],
+                        "substances": [ [ BRICK.measures, BRICK.Return_Air ], [ BRICK.measures, BRICK.Grains ], ],
                     }
                 }
             },
             "Angle_Sensor": {
+                #TODO substances
```

Comment:
Is it still TODO?

Reply:
It is still a TODO. I wasn't sure how we wanted to handle concepts like "position" or "angle". Would you say these are quantities? Or substances?

Reply:
I think both angle and position are quantities. Position of a damper, angle of a fan, etc. In these cases, substances are actually equipment, which I think we discussed a bit about whether we need to allow equipment as tags of sensors for such cases. I think it's worth adding them, but we can discuss that after this release too.

Reply:
Ok cool; I added Angle and Position and associated them with Angle Sensor and Damper Position Sensor. Agreed that this should be a future conversation as well!